tokens/src/zIndex.json

```diff
@@ -0,0 +1,36 @@
+{
+  "z-index": {
+    "below-zero": {
+      "$description": "Stack an element behind elements that do not have a stacking order applied.",
+      "value": "-1"
+    },
+    "zero": {
+      "$description": "Stack an element behind elements that do not have a stacking order applied.",
+      "value": "0"
+    },
+    "above-zero": {
+      "$description": "Stack an element above elements that do not have a stacking order applied, but below elements with other z-indexes applied.",
+      "value": "1"
+    },
+    "tooltip": {
+      "$description": "Use for tooltip components.",
+      "value": "2000"
+    },
+    "flyout": {
+      "$description": "Use for flyout components.",
+      "value": "2100"
+    },
+    "fixed": {
+      "$description": "Use for fixed or sticky elements and fullscreen elements.",
+      "value": "2200"
+    },
```

Comment:
what happens if we have fixed and fullscreen at the same time? Don't we have to precise here?

Reply:
The z-indexes represented here are reusable within any [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context). So, in the scenario of a fullscreen view that contains a fixed header toolbar, we could drop [an `isolate` TW utility class](https://tailwindcss.com/docs/isolation) onto the fullscreen element, and every child element within the fullscreen is magically stacked within that new context.

I've found some occurrences of `z-index: 0;` being (mis)used to establish a new stacking context within demosplan, these may be refactored to use `isolate`, too, to clearly communicate the intention.

Another thought with the small number of z-indexes: i see them as rough guidance on where to place an element. If an element needs a z-index that is "just below the other stuff of that element", say, the arrow of a flyout, the outer element can be positioned by these global z-indexes, while the inner elements can be positioned using a new stacking context.